### DISCUSSION TITLE
Why `bevy_math` doesn't have `Triangle3d` struct?

### DISCUSSION THREAD

**Discussion by busf4ctor** [bevyengine/bevy#12506] [Q&A]
I often find myself needing to work with 3D triangles in one of my personal projects. However, there's no `bevy_math::primitives::Triangle3d`. I believe a dedicated `Triangle3d` struct could be useful.

## Advantages:

- **Clarity:** More explicit representation of a common geometric primitive.
- **Convenience**: Provide methods tailored to triangles (area, normals, etc.).

## Workarounds

I am using the `bevy_math::primitives::polygon`. It sacrifices clarity and convenience.

---

I'm willing to contribute the implementation. Are there any downsides or reasons this might not be a good fit for `bevy_math`?

**Comment by alice-i-cecile** [comment]
I'm interested in this: please open a PR :)
